src/lib/permissions.ts

```diff
@@ -14,3 +14,15 @@ export async function botMasterPerms(msg: Message): Promise<boolean> {
 	const team = msg.client.application.owner as Team;
 	return team.members.has(msg.author.id);
 }
+
+export const STAFF_PERMS: ApplicationCommandPermissionData = {
+	id: ROLES.STAFF,
+	permission: true,
+	type: 'ROLE'
+};
+
+export const ADMIN_PERMS: ApplicationCommandPermissionData = {
```

Comment:
Wouldn't line 8-10 be redundant if this is added? Also, would it be worth adding a bot master perms constant in the same format as this and STAFF_PERMS?

Reply:
8-10 are still there for support of legacy permission functions which I guess aren't necessary anymore. Either way, going to keep until the big yeet:tm:. I don't yet know how to handle botmaster perms, @renross is doing the admin commands so I assumed they would figure it out.